### DISCUSSION TITLE
Creating a User without needing to confirm its email

### DISCUSSION THREAD

**Discussion by odiseo0** [supabase/supabase#7921] [Questions] [answered]
I'm using the Python gotrue API to create users from an admin panel, but there's a problem, the problem is that I create such user but the User needs to confirm its email.

I already have disabled the option of email confirmation, but when a new user is created by an API, it says that 'email confirmation is needed' and Supabase doesn't have a magic confirm button.

This question was asked first in gotrue-py repository as [#135](https://github.com/supabase-community/gotrue-py/issues/135)

**Comment by wcooperscott** [comment, marked as answer]
When creating the account, go ahead and mark it as confirmed with the `email_confirm` attribute.

Here is some code from one of my projects:
`client.auth.api.createUser({
      email,
      password,
      email_confirm: true,
    });`

Hope this helps!

**Reply by odiseo0** [reply]
Thanks, I'll try
